update_version.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+## Usage: ./update_version.sh <version>
+
+set -e
+
+readonly NEW_VERSION=$1
+
+if [ -z "$NEW_VERSION" ]; then
+  echo "You must specify the new version!"
+  exit 1
+fi
+
+# Get the old version from securedrop_log/VERSION
+OLD_VERSION=$(cat securedrop_log/VERSION)
+if [ -z "$OLD_VERSION" ]; then
+  echo "Couldn't find the old version: does this script need to be updated?"
+  exit 1
+fi
+
+# Update the version in securedrop_log/VERSION (setup.py is done automatically)
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    # The empty '' after sed -i is required on macOS to indicate no backup file should be saved.
+    sed -i '' "s@$(echo "${OLD_VERSION}" | sed 's/\./\\./g')@$NEW_VERSION@g" securedrop_log/VERSION
+    sed -i '' "s@$(echo "${OLD_VERSION}" | sed 's/\./\\./g')@$NEW_VERSION@g" securedrop_log/__init__.py
```

Comment:
looks like we don't also need to update `securedrop_log/__init__.py`?

Reply:
Oh, that must be sleepy me and copy paste. :+1: